karma.conf.js

```diff
@@ -20,8 +20,8 @@ module.exports = function(config) {
       'third_party/skulpt-c14015/skulpt.min.js',
       'third_party/skulpt-c14015/skulpt-stdlib.js',
       'third_party/ui-codemirror-0.3.0/ui-codemirror.min.js',
-      'client/app.js',
-      'client/**/*.js',
+      'client/question/question.js',
+      'client/**/**/*.js',
```

Comment:
I think you can leave this as it was, and in fact you don't need the line above either. That's because, in the config file `**/` represents multiple dirs, I think -- could you please check? If so we can just replace this with `client/**/*.js`.